python/pylibcudf/pylibcudf/tests/test_column_from_array.py

```diff
@@ -23,66 +23,81 @@
     np.bool_,
 ]
 
+NUMPY_DTYPES = [
+    *CUPY_DTYPES,
+    *[
+        np.dtype("datetime64[s]"),
+        np.dtype("datetime64[ms]"),
+        np.dtype("datetime64[us]"),
+        np.dtype("datetime64[ns]"),
+        np.dtype("timedelta64[s]"),
+        np.dtype("timedelta64[ms]"),
+        np.dtype("timedelta64[us]"),
+        np.dtype("timedelta64[ns]"),
```

Comment:
Note: I'm not testing timezone-aware datetimes 
> This is a “naive” time, with no explicit notion of timezones or specific time scales (UT1, UTC, TAI, etc.).
https://numpy.org/doc/stable/reference/arrays.datetime.html#datetime64-conventions-and-assumptions